src/Microsoft.IdentityModel.Abstractions/ITelemetryClient.cs

```diff
@@ -63,15 +63,15 @@ public interface ITelemetryClient
         /// <param name="stringProperties">Key value pair of strings to long with the event.</param>
         /// <param name="longProperties">Key value pair of longs to long with the event.</param>
         /// <param name="boolProperties">Key value pair of bools to long with the event.</param>
-        /// <param name="DateTimeProperties">Key value pair of DateTimes to long with the event.</param>
+        /// <param name="dateTimeProperties">Key value pair of DateTimes to long with the event.</param>
         /// <param name="doubleProperties">Key value pair of doubles to long with the event.</param>
         /// <param name="guidProperties">Key value pair of Guids to long with the event.</param>
         void TrackEvent(
             string eventName,
             IDictionary<string, string> stringProperties = null,
             IDictionary<string, long> longProperties = null,
             IDictionary<string, bool> boolProperties = null,
-            IDictionary<string, DateTime> DateTimeProperties = null,
+            IDictionary<string, DateTime> dateTimeProperties = null,
             IDictionary<string, double> doubleProperties = null,
```

Comment:
[src/Microsoft.IdentityModel.Abstractions/NullTelemetryClient.cs](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/f3ddfe0dcb40ec5bde3e324b4246984753a0e121/src/Microsoft.IdentityModel.Abstractions/NullTelemetryClient.cs) would also need the same change.

Reply:
Good catch thanks Sruthi